examples/backends/vllm/launch/xpu/agg_omni_i2v_xpu.sh

```
@@ -0,0 +1,85 @@
+#
+# Usage:
+#   bash agg_omni_i2v.sh [OPTIONS]
+#
+# Options:
+#   --model <model>   Model to use (default: Wan-AI/Wan2.2-TI2V-5B-Diffusers)
+#   Any other flags are forwarded to the vLLM worker.
+set -e
+trap 'echo Cleaning up...; kill 0' EXIT
+
+SCRIPT_DIR="$(dirname "$(readlink -f "$0")")"
+source "$SCRIPT_DIR/../../../../common/launch_utils.sh"
```

Shellcheck (0.11.0)

[info] Not following: ./../../../../common/launch_utils.sh was not specified as input (see shellcheck -x).

(SC1091)